model/semconv/v1.6.1/nonstandard.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semconv
+
+const (
+	InstrumentationLibraryName    = "otel.library.name"
+	InstrumentationLibraryVersion = "otel.library.version"
```

Comment:
To ensure consistency I think these should be named "OtelLibraryName" and "OtelLibraryVersion".

Reply:
Should I change these in the `v1.5.0` package as well?

Reply:
Not sure :D waiting for suggestion

Reply:
On further review, `InstrumentationLibraryName` and `InstrumentationLibraryVersion` are the correct identifiers.  These are not used to identify an API or SDK version, but for an instrumentation library to identify itself when creating a `TracerProvider` or `MeterProvider`.  See [OTEP-83](https://github.com/open-telemetry/oteps/blob/main/text/0083-component.md).

Reply:
But they are not following the same semantic of constructing the names based on the value

Reply:
I worry that this may lead to confusion regarding the purpose of this attribute, but have changed the name.  It will need to be accounted for when updating usage from `1.5.0` to `1.6.1`.